samples/apps/copilot-chat-app/webapp/src/redux/features/conversations/ChatState.ts

```diff
@@ -11,4 +11,11 @@ export interface ChatState {
     botProfilePicture: string;
     lastUpdatedTimestamp?: number;
     input: string;
+    isBotTyping: boolean;
```

Comment:
Why are we treating the bot separately from the user? 

Bot is a user and can be identified by the UserRole

Reply:
Two reasons:
1. The bot is currently not registered as a ChatUser in the WebApp.
2. The typing signal broadcast as a boolean. We can in the future create a new model called UserState and broadcast that instead.